plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Pom.java

```diff
@@ -270,6 +280,66 @@ void updateDependencies(Collection<Library> selectedLibraries,
     }   
   }
 
+  private void createBOMIfNeeded(XPath xpath) throws CoreException {
+    try {
+      Element bomElement = (Element) xpath.evaluate(
+          "//m:dependencyManagement/m:dependencies/m:dependency[m:groupId='"
+              + Bom.GOOGLE_BOM_GROUP_ID + "'][m:artifactId='" + Bom.GOOGLE_BOM_ARTIFACT_ID + "']",
```

Comment:
Now that I think about it I wouldn't ad these constants to the Bom clkass either. They add nothing to the string literals.